src/Serializer/AbstractItemNormalizer.php

```diff
@@ -417,7 +489,7 @@ protected function getAttributeValue($object, $attribute, $format = null, array
      *
      * @return string|array
      */
-    private function normalizeRelation(PropertyMetadata $propertyMetadata, $relatedObject, string $resourceClass, string $format = null, array $context)
+    protected function normalizeRelation(PropertyMetadata $propertyMetadata, $relatedObject, string $resourceClass, string $format = null, array $context)
```

Comment:
Sure. I reverted most of the changes, sorry about that.

This one I needed it because the JSON API item normalizer returns an [array with `type` and `id`](http://jsonapi.org/format/#document-resource-identifier-objects) instead of an IRI.